src/wp-content/themes/twentynineteen/inc/template-functions.php

```diff
@@ -163,7 +163,7 @@ function twentynineteen_add_ellipses_to_nav( $nav_menu, $args ) {
  *
  * @ref https://www.w3.org/WAI/tutorials/menus/flyout/
  */
-function twentynineteen_nav_menu_link_attributes( $atts, $item, $args, $depth ) {
+function twentynineteen_nav_menu_link_attributes( $atts, $item ) {
```

Comment:
While touching this function, maybe consider adding the `@param` to the function's docblock.

Reply:
The feedback addressed in https://github.com/WordPress/wordpress-develop/commit/dec4b321f99acf4c62dfe2d9f7e2513ebeaf26e3

Reply:
Nitpick: Because this PR already improves the docblock, I would like the "tot" corrected to "to" here instead of on another ticket.